JPS_VIRTUALSENSOR/AermodAgent/src/main/java/com/cmclinnovations/aermod/QueryClient.java

```diff
@@ -970,8 +970,15 @@ public void setElevation(List<StaticPointSource> pointSources, List<Building> bu
 
             try (Statement stmt = rdbStoreClient.getConnection().createStatement()) {
```

Comment:
be sure to check if this closes the connection, I'm not 100% sure if closing the statement closes the connection

Reply:
I added a finally clause to close the connection. Please see commit #bce18e2.

Reply:
I updated that part to use try-with resources to close the connection.